backend/sozisel/lib/sozisel/model/quizzes/quiz.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Sozisel.Model.Quizzes.Quiz do
+  use Sozisel.Model.Schema
+  import Ecto.Changeset
+
+  alias Sozisel.Model.Quizzes.QuizQuestion
+
+  @type t :: %__MODULE__{
+          duration_time: Integer.t(),
+          number_of_targets: Integer.t(),
+          tracking_mode: Boolean.t(),
+          quiz_questions: [QuizQuestion]
+        }
+
+  @primary_key false
+
+  embedded_schema do
+    field :duration_time, :integer
```

Comment:
Duration time is supposed to be in minutes, seconds?

Reply:
in seconds